.github/workflows/test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         - "test.run.test_noop"
         - "test.run.test_sources"
         - "test.run.test_stages"
+        - "-k stages/test"
```

Comment:
Note that because `stages/__init__.py` and `stages/test/__init__.py` are missing it is not possible to use the existing style of just doing `stages.test`. Adding `stages/__init__.py` feels wrong and the desire is to have the stages tests close to the stages so this seems the least invasive way.

But happy to reconsider/adjust if you prefer something else.

Reply:
This is ok :)